rai/core_test/wallets.cpp

```diff
@@ -2,12 +2,28 @@
 
 #include <rai/node/testing.hpp>
 
-TEST (wallets, DISABLED_open_create)
+void destroy_existing_wallets (rai::wallets & wallets)
+{
+	// We can't change a map while iterating over it,
+	// so we make a copy of the keys we need to destroy.
+	std::vector<rai::uint256_union> to_delete;
+	for (auto it : wallets.items)
+	{
+		to_delete.push_back (it.first);
+	}
+	for (auto it : to_delete)
+	{
+		wallets.destroy (it);
+	}
+}
+
+TEST (wallets, open_create)
 {
 	rai::system system (24000, 1);
 	bool error (false);
 	rai::wallets wallets (error, *system.nodes[0]);
 	ASSERT_FALSE (error);
+	destroy_existing_wallets (wallets);
```

Comment:
This is a case where the expectation of the test is wrong, you should just change the assertion. The intent is to test the behavior of rai::wallets, not test new destroy code.

Reply:
Makes sense. I just wasn't sure what was creating the wallet, and as such, I didn't want to rely on finicky behavior. I'll update this to the new behavior.

Reply:
If the wallet has flaky behavior, that should definitely get fixed (and maybe another test added to further expose the bad behavior).

Reply:
I'm pretty sure it doesn't, but at first, I thought another test was creating the wallet. I didn't realize they were independent at first.